src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/adjudications/AdjudicationsMigrationService.kt

```diff
@@ -79,17 +79,21 @@ class AdjudicationsMigrationService(
           )
 
         adjudicationsService.createAdjudication(nomisAdjudication.toAdjudication())
+        val chargeNumber =
+          "${nomisAdjudication.adjudicationNumber}/$chargeSequence" // TODO: returned by DPS but assume it made from these fields
         createAdjudicationMapping(
           adjudicationNumber = adjudicationNumber,
           chargeSequence = chargeSequence,
-          chargeNumber = "${nomisAdjudication.adjudicationNumber}/$chargeSequence", // TODO: returned by DPS but assume it made from these fields
+          chargeNumber = chargeNumber,
           context = context,
         )
 
         telemetryClient.trackEvent(
           "adjudications-migration-entity-migrated",
           mapOf(
             "adjudicationNumber" to adjudicationNumber.toString(),
+            "chargeSequence" to chargeSequence.toString(),
+            "chargeNumber" to chargeNumber,
             "migrationId" to context.migrationId,
           ),
```

Comment:
interestingly we don't have  test for telemetry since we check the bulk telemetry (start, complete etc).
I'm thinking we need an integration test that better describes the behaviour of a single item being migrated with some better asserts